README.md

```diff
@@ -9,29 +9,24 @@
 
 Install python dependencies:
 
+__Ubuntu 18.04__, __Ubuntu 20.04__, __Linux Mint 20__
+
 ```sh
 sudo apt install python3-pip
 pip3 install -r requirements.txt
 ```
 
-__Ubuntu 18.04__ and __Ubuntu 20.04__
+__Linux Mint 19__
 
 ```sh
-sudo apt install librocksdb-dev libglib2.0-dev qtbase5-dev libboost-system-dev libboost-filesystem-dev libboost-program-options-dev make cmake antlr gcc g++ llvm-7 libcurl4-openssl-dev libclang-7-dev libboost-python-dev
-```
-
-Install Python development package
```

Comment:
Not required to do manually as it is installed as a dependency of libboost-python-dev.